model/listeners.py

```diff
@@ -69,7 +69,11 @@ def _site_configuration_has_changed(_db, cooldown=1):
         # Update the timestamp.
         now = utc_now()
         earlier = now - datetime.timedelta(seconds=cooldown)
-        sql = "UPDATE timestamps SET finish=(:finish at time zone 'utc') WHERE service=:service AND collection_id IS NULL AND finish<=(:earlier at time zone 'utc');"
+        sql = (
```

Comment:
Need to add a comment here with a bit more detail why we are using a more complicated query to prevent deadlocks.

Reply:
Comment added